pepsico/globals_.py

```diff
@@ -0,0 +1,22 @@
+import flask
+import pingrid
+import os
+
+defaultconfig = pingrid.load_config("config-defaults.yaml")
+appconfig = pingrid.load_config(os.environ["CONFIG"])
+GLOBAL_CONFIG = pingrid.deep_merge(
+    {k: v for k, v in defaultconfig.items() if k != "maprooms"},
+    {k: v for k, v in appconfig.items() if k != "maprooms"},
+)
+GLOBAL_CONFIG['maprooms'] = {}
+for k, v in appconfig['maprooms'].items():
+    if isinstance(v, list):
+        GLOBAL_CONFIG['maprooms'][k] = [pingrid.deep_merge(defaultconfig['maprooms'][k], v[i]) for i in range(len(v))]
+    elif v is not None:
+        GLOBAL_CONFIG['maprooms'][k] = pingrid.deep_merge(defaultconfig['maprooms'][k], v)
+
+
+FLASK = flask.Flask(
+    "pepsicomaprooms",
+    static_url_path=f'{GLOBAL_CONFIG["url_path_prefix"]}/static',
+)
```

Comment:
Are there going to be multiple maprooms eventually? If not, we can simplify the setup.

Reply:
For the sake of time, we are making a Global one now, but they might want one per "Region" of interest and I am not sure if they'd want that as a control on the Maproom page, or as as many separate applications (which on another hand could also help with performance...). Then if the collaboration continues and they ask for new Maprooms that don't yet exist in classic style, then more Python ones should come up too.